packages/api-client/src/user/data/UserPropertiesSetData.ts

```diff
@@ -32,10 +32,6 @@ export enum NotificationPreference {
 
 export interface WebappProperties {
   [property_key: string]: any;
-  contact_import: {
-    google?: number;
-    macos?: number;
-  };
   enable_debugging: boolean;
```

Comment:
I am wondering if these properties should be optional?

Reply:
Let's remove them.

Reply:
I meant the other properties like `enable_debugging`.

Reply:
Does it cause any problems having them to be mandatory?

Reply:
For now it doesn't :)